content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started with Entity Framework Core
```

Comment:
This file should have the table of contents too. Sorry. I know it's duplicate data, but that is how they have it set up.